src/main/java/com/untamedears/itemexchange/rules/modifiers/_ExampleModifier.java

```diff
@@ -15,17 +15,10 @@
 
 @CommandAlias(SetCommand.ALIAS) // This is needed to make commands work
 @Modifier(slug = "EXAMPLE", order = 12345)
-public final class _ExampleModifier extends ModifierData<_ExampleModifier> {
+public final class _ExampleModifier extends ModifierData {
```

Comment:
The heck is with that class name

Reply:
This class isn't used at all, it's just an example, and the `_` is to ensure it's at the top of the folder. When creating a new Modifier, I just copy paste this class and rename it.